File: tags/Versión Final (antes de CC)/Cliente/Cliente/Transformadores/EnteroMinimoABlanco.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Data;

namespace Trascend.Bolet.Cliente.Transformadores
{
    class EnteroMinimoABlanco : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
        {
            int numero =  int.Parse(value.ToString());

            return numero.Equals(int.MinValue) ? "" : numero.ToString(); 
        }

        public object ConvertBack(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
        {
            throw new NotImplementedException();
        }
    }
}

File: trunk/src/Cliente/Cliente_Fac/Transformadores/EnteroMinimoABlanco.vb
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows.Data
Imports System.Windows.Controls
Namespace Transformadores
    Class EnteroMinimoABlanco
        Implements IValueConverter

        Public Function Convert(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements System.Windows.Data.IValueConverter.Convert
            Dim numero As Integer = Integer.Parse(value.ToString())

            Return If(numero.Equals(Integer.MinValue), "", numero.ToString())
        End Function

        Public Function ConvertBack(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements System.Windows.Data.IValueConverter.ConvertBack
            Throw New NotImplementedException()
        End Function

    End Class
End Namespace
